### PR TITLE
Remove more dialogue matching code

### PR DESCRIPTION
**Pull request by rjmk**
These resolvers were only used for the now-removed Dialogue Matching page

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1208173768368792) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/lib/generated/fragmentTypes.d.ts [hunk 1]
```diff
@@ -890,7 +890,6 @@ interface PostsDefaultFragment { // fragment on Posts
   readonly tableOfContentsRevision: any,
   readonly sideComments: any,
   readonly sideCommentVisibility: string,
-  readonly hasFootnotes: boolean,
```
Comment (rjmk) [inline packages/lesswrong/lib/generated/fragmentTypes.d.ts:893]: I dunno why this change is here

**Review by oetherington — COMMENTED** [review]
No objections from EAF

File: packages/lesswrong/lib/generated/fragmentTypes.d.ts (hunk 1 above)
Comment (oetherington) [reply packages/lesswrong/lib/generated/fragmentTypes.d.ts:893]: Looks like someone merged into master without running `yarn generate` (nothing major)

**Comment by b0b3rt** [comment]
@rjmk I think we can now get rid of a bunch of repo methods that were only used by these resolvers, too?

**Comment by rjmk** [comment]
Repo excoriated, @b0b3rt

**Review by b0b3rt — APPROVED** [review]
🎉
